server/server_start.py

```diff
@@ -0,0 +1,6 @@
+from server.lib.network_interface import *
+import server.lib.request_handler as req
+
+if __name__ == "__main__":
+    __testChain = req.EchoRequestHandler(req.PingRequestHandler(req.ServerListRequestHandler()))
+    __server = Server(__testChain)
```

Comment:
Tu te chainy trzeba wyrzucić chyba że wybierzesz tą opcje ze ściągnięciem pliku servera z developa to ten plik uruchomienia wgl nie potrzebny bo tam już uruchomienie jest